src/App.css

```diff
@@ -51,18 +52,45 @@
   text-align: right;
   color: white;
   margin-top:0px;
-  padding:30px 35px 20px 25px;
+  padding:10px 35px 15px 25px;
   background: rgb(58, 56, 56);
   font-size: 15px;
-  border-bottom: 7px solid #500966;
+  border-bottom: 7px solid #b5e77d;
 }
 
 .footer {
-  margin-top:100px;
   text-align: center;
   color: purple;
-  padding:1px 35px 50px 25px;
+  padding:0px 35px 50px 25px;
   background: rgb(58, 56, 56);
   font-size: 15px;
-  border-top: 7px solid #500966;
+  border-top: 7px solid #b5e77d;
+}
+
+h1 {
+  margin-top: 0;
+  padding-top: 10px;
+  background: white;
+  margin-bottom: 0;
+}
+
+h2 {
+  margin-top: 0;
+  padding-top: 30px;
+  padding-bottom: 25px;
+  background: white;
+  margin-bottom: 0;
+}
```

Comment:
Use Typography elements instead. We can customize what these are in a theme later.

Reply:
Ok I will be using Typography and the CSS file no longer has any of this.